### ISSUE TITLE
"nix-shell" requires "bash" runtime dependency

### ISSUE THREAD

**Issue by massimo-zaniboni** [NixOS/nixpkgs#9393]
I installed Nix on a Linux Mint OS. If I execute something like

```
nix-shell '<nixpkgs>' -A hello 
```

the shell is not started, no error message is displayed, and error code is 0 (no error).

If I install bash, the nix-shell works correctly:

```
nix-env -i bash 
nix-shell '<nixpkgs>' -A hello 
```

Probably nix-shell requires some explicit runtime dependency to the bash package, but I don't know how to specify it in nix-shell script.

**Comment by FRidh** [comment]
I think that, if you need bash and its not a dependency, you need to explicitly add bash yourself.
`nix-shell '<nixpkgs>' -A hello -p bash`

**Comment by massimo-zaniboni** [comment]
@FRidh nix-shell needs bash package for working correctly, so nix-shell can depend explicitly from bash package, or Nix can install bash package by default during initial installation, or the manual in the nix-shell section, should specify that installing bash package is mandatory.

I'm saying this, because I spent a lot of time trying nix-shell examples. There were no error messages, but only some strange behavior that puzzled me. It started working correctly only when I installed bash package.
